website/docs/fun/anon-channel/component-handler.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Component Handler
+---
+
+This command responds to the "Send an anonymous message" button created by the main cc by presenting a modal for users
+to send their message in.
+
+For more information about the anonymous channel, see [this](overview) page.
+
+## Trigger
+
+**Type:** `Component`<br />
+**Custom ID:** `\Aanon_submit\b`
+
+## Configuration
+
+See the [anonymous channel overview](overview/#configuration) for in-depth information about configuration.
```

Comment:
```suggestion
```
As there's no configuration necessary for this sub-CC, no need for this header; even less so if it just points to the main CC.